extensions/resource-deployment/notebooks/bdc/2019/deploy-bdc-existing-kubeadm.ipynb

```diff
@@ -237,7 +237,7 @@
         {
             "cell_type": "code",
             "source": [
-                "run_command(f'azdata login --cluster-name {mssql_cluster_name}')"
+                "run_command(f'azdata login -n {mssql_cluster_name}')"
```

Comment:
>mssql_cluster_name [](start = 48, length = 18)

So currently we allow spaces in the name which is likely going to break this. We should make sure to handle that appropriately (not necessarily in this PR)

Reply:
according to this: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/ , space is not allowed, I need to add input validation for all fields. tracked here: #7612